src/api/transaction/transactionUtils.ts

```diff
@@ -84,6 +88,17 @@ export async function readArrowFiles(files: TransactionAsyncFile[]) {
   return results;
 }
 
+export async function readProtoMetadata(file: File | Blob) {
+  if (!file?.arrayBuffer) {
+    throw new Error(`Unsupported metadata type: ${typeof file}`);
+  }
```

Comment:
Do we still need the error handling?  In which case would either the `File` or `Blob` type not contain the array buffer?

Reply:
Yes, I think so. We had an issue when `node-fetch` was returning a string instead of `File | Blob`

Reply:
that's fixed in the engines now, but i guess there could always be an old engine lurking out there 👀 
makes sense, thanks